dev/tasks/conda-recipes/r-arrow/bld.bat

```diff
@@ -1,9 +1,14 @@
-bash %RECIPE_DIR%/build_win.sh
-IF %ERRORLEVEL% NEQ 0 exit 1
-cp %RECIPE_DIR%/configure.win r
-IF %ERRORLEVEL% NEQ 0 exit 1
-cp %RECIPE_DIR%/install.libs.R r/src
-IF %ERRORLEVEL% NEQ 0 exit 1
-set "MAKEFLAGS=-j%CPU_COUNT%"
-"%R%" CMD INSTALL --build r
-IF %ERRORLEVEL% NEQ 0 exit 1
+@echo on
+
+bash %RECIPE_DIR%/build_win.sh
+IF %ERRORLEVEL% NEQ 0 exit 1
```

Comment:
This file had mangled line endings, see https://github.com/apache/arrow/pull/14832/commits/115fe7573902adb30f60a1b39c2eaa3759fdb349